packages/generators/generators/lib/plops/prompts/get-attributes-prompts.js

```diff
@@ -92,6 +92,10 @@ module.exports = async inquirer => {
 
   if (addAttributes) {
     await createNewAttributes(inquirer);
+  } else {
+    console.warn(
```

Comment:
Shouldn't we just put the warning in the question or right after the answer ? Feels like the warning is coming a bit later-

Reply:
You're right, maybe something more like a confirmation?

Reply:
I would keep it simple with a simple info in parenthesis in the question or right after the user says yes or no

Reply:
The way it is currently implemented is right after the user says yes or no, but we could move it directly below the question if you'd like

Reply:
@JabStrapi  what do you think ?

Reply:
I'll go with your first version and we can update it later